### PR TITLE
feat : add grid component

### PR DESCRIPTION
**Pull request by Omkar-Chaskar**
add grid component with grid containing 2 items and 3 items.

### CODE REVIEW

File: Components/grid.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    display: grid;
+    padding: 5px;
+}
+
+.col-2 {
+    grid-template-columns: auto auto;
+  }
+  
+.col-3 {
+    grid-template-columns: auto auto auto;
```
Comment (sushil-mod) [inline Components/grid.css]: this is also correct , you can also try repeat(3,1fr) (just a suggestion )

**Review by sushil-mod — APPROVED** [review]
just a suggestion . overall short and clean code

File: Components/grid.css (hunk 1 above)
Comment (Omkar-Chaskar) [reply Components/grid.css]: Thanks for suggestion  , I will implement it.

File: Components/grid.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+.col-2 {
+    grid-template-columns: repeat(2, auto);
+  }
+  
+.col-3 {
+    grid-template-columns: repeat(3, auto);
+  }
+  
+
```
Comment (Kothalkarpriya) [inline Components/grid.css]: remove the extra spaces

File: documentation.html [hunk 3]
```diff
@@ -1073,6 +1074,65 @@ <h3 class="doc-header">Snackbar</h3>
...
+                                </div>
+                                <div class="card__media-row">
+                                  <img class="card-img-eg" src=https://images.unsplash.com/photo-1606913852449-8ebf553565cf?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHx0b3BpYy1mZWVkfDE1OHx0b3dKWkZza3BHZ3x8ZW58MHx8fHw%3D&auto=format&fit=crop&w=500&q=60; alt="">
+                                </div>
+                                <div class="card__primary card-hover">
+                                    <h2 class="card__title">Our Changing Planet</h2>
+                                    <h3 class="card__subtitle">by Kurt Wagner</h3>
+                                  </div>
+                        </div>
+                            </div>
```
Comment (Kothalkarpriya) [inline documentation.html:1114]: use code formatting

**Review by Kothalkarpriya — APPROVED** [review]
Do look into my suggestions once and then you can merge

File: Components/grid.css [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+.col-2 {
+    grid-template-columns: repeat(2, auto);
+  }
+  
+.col-3 {
+    grid-template-columns: repeat(3, auto);
+  }
+
```
Comment (Kavita539) [inline Components/grid.css]: Remove extra spaces if not needed.

**Review by Kavita539 — APPROVED** [review]
- Code is clean and readable.
- Even UI looks good and minimal.
- Just a minor suggestion and overall looks good.

File: Components/grid.css [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  
+.col-3 {
+    grid-template-columns: auto auto auto;
+  }
+  
+
+  /*Card example*/
+  .card-eg {
+    width : min-content;
+    height : max-xontent;
```
Comment (kedark152) [inline Components/grid.css]: replace x with c in height: max-content, spelling mistake

**Pull request merged**